helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -173,7 +173,7 @@ receivers:
       {{- if .Values.agent.controlPlaneMetrics.proxy.enabled }}
       smartagent/kubernetes-proxy:
         {{- if eq .Values.distribution "openshift" }}
-        rule: type == "pod" && labels["app"] == "sdn"
+        rule: type == "port" && pod.labels["app"] == "sdn" && (port == 9101 || port == 29101)
```

Comment:
Just confirming, only one of these ports is available on proxy pod? or we'll have 2 receiver per proxy

Reply:
Only one port will be used. I've only seen port 29101 used in versions 4.9 and below. Newer versions use 9101.